src/scripts/services/client-service.ts

```diff
@@ -43,3 +46,23 @@ export const oktaManageClient = (
     authorizationMode: 'PrivateKey',
     scopes: scopes.map((scope) => 'okta.' + scope + '.manage'),
   });
+
+/**
```

Comment:
**issue:** move to `okta-service.ts` or similar please. `TokenEndpointResponse` is part of the Okta SDK internals (in my view) and shouldn't escape from the service - change this to return `true` if the credentials are valid.

Reply:
modified in commit: 9eeee944340adbabb37bd98f13005b40bf462248